### DISCUSSION TITLE
计算机科学速成课 | hackway.org

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#27] [Announcements]
# 计算机科学速成课 | hackway.org

课程名称： Crash Course Computer Science

https://hackway.org/docs/cs/freshman/first/csintro#comments

**Comment by dgdjelly** [comment]
您好，请问这些课程是都得学习麻？

**Reply by lidongyx** [reply]
你可以选择一个学校的学习路线来循序渐进地学习，而每一门课最好都学习一下
